src/renderer/containers/Settings/DolphinSettings.tsx

```diff
@@ -28,12 +34,11 @@ export const DolphinSettings: React.FC<{ dolphinType: DolphinLaunchType }> = ({
   );
   const [dolphinPath, setDolphinPath] = useDolphinPath(dolphinType);
   const [resetModalOpen, setResetModalOpen] = React.useState(false);
-  const [isResetting, setIsResetting] = React.useState(false);
+  const [isResetting, setIsResetting] = React.useState(ResetType.NONE);
```

Comment:
```suggestion
  const [isResetType, setResetType] = React.useState(ResetType.NONE);
```
This variable needs to be renamed too.